client/js/store/map/ProcedureMapSettings.js

```diff
@@ -39,9 +39,10 @@ export default {
         .then(response => checkResponse(response))
     },
 
-    fetchProcedureMapSettings ({ commit }, procedureId) {
+    fetchProcedureMapSettings ({ commit }, args= { procedureId, isMaster }) {
```

Comment:
s there be an `args` argument or can we skip it and just use `{ procedureId, isMaster }`? It would be shorter

Reply:
I could not make it work otherwise.